orientdb/src/test/java/org/hibernate/datastore/ogm/orientdb/util/MemoryDBUtil.java

```diff
@@ -0,0 +1,131 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ * 
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.datastore.ogm.orientdb.util;
+
+import com.orientechnologies.orient.core.id.ORecordId;
+import com.orientechnologies.orient.core.metadata.schema.OProperty;
+import com.orientechnologies.orient.core.metadata.schema.OType;
+import com.orientechnologies.orient.core.sql.OCommandSQL;
+import com.tinkerpop.blueprints.Vertex;
+import com.tinkerpop.blueprints.impls.orient.OrientGraphFactory;
+import com.tinkerpop.blueprints.impls.orient.OrientGraphNoTx;
+import com.tinkerpop.blueprints.impls.orient.OrientVertexType;
+import java.math.BigDecimal;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * @author Sergey Chernolyas <sergey.chernolyas@gmail.com>
+ */
+public class MemoryDBUtil {
+
+	private static final Logger LOG = Logger.getLogger( MemoryDBUtil.class.getName() );
+	private static OrientGraphFactory factory;
+
+	@Deprecated
+	private static void addCommonProperties(OrientGraphNoTx graph, OrientVertexType type) {
```

Comment:
Why `@Deprecated`?
This part has not been included yet. You can remove it or change it the way you prefer